src/coreclr/tools/Common/Compiler/DependencyAnalysis/Target_X64/TargetRegisterMap.cs

```diff
@@ -20,30 +20,11 @@ public struct TargetRegisterMap
 
         public TargetRegisterMap(TargetOS os)
         {
-            switch (os)
-            {
-                case TargetOS.Windows:
-                    Arg0 = Register.RCX;
-                    Arg1 = Register.RDX;
-                    Arg2 = Register.R8;
-                    Arg3 = Register.R9;
-                    Result = Register.RAX;
-                    break;
-
-                case TargetOS.Linux:
-                case TargetOS.OSX:
-                case TargetOS.FreeBSD:
-                case TargetOS.SunOS:
-                case TargetOS.NetBSD:
-                    Arg0 = Register.RDI;
-                    Arg1 = Register.RSI;
-                    Arg2 = Register.RDX;
-                    Arg3 = Register.RCX;
-                    Result = Register.RAX;
-                    break;
-                default:
-                    throw new NotImplementedException();
-            }
+            Arg0 = os == TargetOS.Windows ? Register.RCX : Register.RDI;
```

Comment:
Nit: I find the switch more readable. I can imagine someone doing a pass on this with some tool in the future and delete the extra whitespace that is used to align things to be readable right now. People love running analyzers, change things and make them subtly less readable for the sake of consistentency...

I would just change `case TargetOS.Blah:` to `default:`

Reply:
Agreed. Done.